src/content/1.7/development/webservice/tutorials/advanced-use/manage-configuration.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Manage Configuration
+menuTitle: Configuration
+weight: 4
+---
+
+# Manage Configuration
+
+You can manage your shop configuration thanks to the API, in this example we will set the `PS_MULTISHOP_FEATURE_ACTIVE` to true (which enables multishop mode).
+
+## API call
+
+First check if the configuration already exists by calling `/api/configurations/?display=[id,name,value]&filter[name]=[PS_MULTISHOP_FEATURE_ACTIVE]`
+
+If the configuration doesn't exist yet, you need to POST on the `configuration` API `/api/configurations/`
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<prestashop xmlns:xlink="http://www.w3.org/1999/xlink">
+    <configuration>
+        <value>1</value>
+        <name>PS_MULTISHOP_FEATURE_ACTIVE</name>
+    </configuration>
+</prestashop>
+```
+
+If it is already defined you need to update it via a PUT using the configuration ID: `/api/configurations/411`
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<prestashop xmlns:xlink="http://www.w3.org/1999/xlink">
+    <configuration>
+        <id>411</id>
+        <value>1</value>
+        <name>PS_MULTISHOP_FEATURE_ACTIVE</name>
+    </configuration>
+</prestashop>
+```
+
+{{% notice warning %}}
+The API being a simple CRUD it lacks logic on resources and doesn't check for existing values, so be sure to check the existence of the configuration and use the appropriate action `POST|PUT` if you want to avoid duplicates.
+{{% /notice %}}
+
+## PHP Webservice lib
+
+```php
+<?php
+
+require_once('./vendor/autoload.php');
+
+try {
+    $webServiceUrl = 'http://example.com/';
+    $webServiceKey = 'ZR92FNY5UFRERNI3O9Z5QDHWKTP3YIIT';
```

Comment:
```suggestion
    $webServiceKey = 'ABCDEFGHIKLMNOPQRSTUVWXYZ0123456';
```

Reply:
I used the same sample key everywhere in the rest of the doc,